src/main/java/com/mael/ttt/Mark.java

```diff
@@ -14,4 +15,10 @@
     public String getString() {
         return mark;
     }
+
+    public Mark swapMark() {
+        if (this == EMPTY)
```

Comment:
This is the place where you notice that "Mark" as the content of your board may be a little "off-by-one": On the one hand, you need to represent _the fact that a location is empty_, on the other hand you want to swap players, but only the X and O ones.

My ultimate favourite variation of this would be Optional<Mark> and Mark is only PLAYER and OPPONENT. That would be a great next step after this PR :smile: